frontend/package.json

```diff
@@ -37,25 +37,25 @@
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.901.4",
-    "@angular/cli": "~9.1.4",
-    "@angular/compiler": "~9.1.4",
-    "@angular/compiler-cli": "~9.1.4",
-    "@angular/language-service": "~9.1.4",
+    "@angular-devkit/build-angular": "^0.901.7",
+    "@angular/cli": "^9.1.7",
+    "@angular/compiler": "^9.1.9",
+    "@angular/compiler-cli": "^9.1.9",
+    "@angular/language-service": "^9.1.9",
```

Comment:
I am wondering why npm update changed this from `~` to `^`. I think it would be better, for all entries, to have just the patch versions applied... Could you maybe manually change the entries back to be prefixed with `~`?

Reply:
yep i will change this tomorrow :)